posthog/caching/test/test_tolerant_zlib_compressor.py

```diff
@@ -51,13 +52,19 @@ def test_the_zlib_compressor_compression(self, _, setting: bool, input: bytes, o
             (
                 "test_when_enabled_can_decompress",
                 True,
-                compressed_bytes,
+                zlib_compressed_bytes,
```

Comment:
nice... this seems the critical bit... we can handle zlib _and_ zstd _and_ uncompressed so we're safe as the data rotates through 🙌